openmdao/docs/features/core_features/working_with_derivatives/checking_partials.rst

```diff
@@ -126,6 +126,20 @@ However, we choose here to use this setting for all comps.
     openmdao.core.tests.test_check_derivs.TestCheckPartialsFeature.test_set_step_calc_global
     :layout: interleave
 
+Includes and Excludes
+---------------------
+
+When you have a model with a large number of components, you may want to reduce the number of components you
+check so that the output is small and readable. The `check_partials` method has two arguments: "inputs" and
+"outputs" that help you specify a reduced set. Both of these arugments are lists that default to None. If you
+specify "includes", and give it some strings, then only the components whose full pathnames match those strings
+are valid. Wildcards are acceptable. Likewise, if you specify excludes, then components whose pathname matches
+the given patterns will be included. You can use both arguments together.
```

Comment:
`included` -> `excluded`
also, suggest `valid` -> `included` in the previous line

Reply:
mmm.. I see now "matches the given patterns" may have been intended to mean the combination of the includes and excludes...    so maybe it's just not clear to those with limited reading comprehension ;)

Reply:
I've reworded things a little bit so that people of all levels can understand.